Automattic/pocket-casts-android
# [Project] Manage Downloaded Episodes  - Update Pencil Icon

## Description
- This PR just updates the pencil icon that had a wrong size after exporting from Figma
- We asked here to the designs and they provided the correct icon: p1730750551749859/1730378068.922729-slack-C05RR9P9RAT


## Testing Instructions

This PR just updates the icon, but if you want to see the icon you can follow these steps:

1. Apply [mock-low-storage.patch](https://github.com/user-attachments/files/17647945/mock-low-storage.patch)
2. Download an episode
3. Go to Downloads
4. See the pencil icon in the banner
5. Go to Profile -> Settings -> Storage screen
6.  See the pencil icon in the modal bottom sheet


## Screenshots or Screencast 

| Modal | Banner |
|--------|--------|
| ![modal](https://github.com/user-attachments/assets/fc9b9e2e-c849-4afe-8c9a-e85d8a70f867) | ![banner](https://github.com/user-attachments/assets/1fe60a7b-66af-4ef3-868a-2be29bece55e) | 






## Checklist
 
#### I have tested any UI changes...
